projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -33,7 +33,7 @@
 @Slf4j
 public class JobImageDeployer {
 
-  // TODO: Out of the image we pick only the python vdk module
+  // TODO: Out of the image we pick only the python vdk module. Remove when property deprecated
```

Comment:
nit: what's this todo? is there a ticket for it, or plan in the specification about it?

Reply:
There is an issue about it, https://github.com/vmware/versatile-data-kit/issues/1929